src/components/ColorPicker/ColorPicker.stories.tsx

```diff
@@ -1,17 +1,53 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { Meta, Story } from "@storybook/react";
-import { EXAMPLE_PALETTES } from "@utilities/colors";
 import React, { useState } from "react";
-import { ColorFormat } from "../../types/colors";
+import tinycolor from "tinycolor2";
+import { Color, ColorFormat, Palette } from "../../types/colors";
 import { ColorPicker, ColorPickerProps } from "./ColorPicker";
 
+const generatePalette = (color: string, amount: number): Color[] => {
+    const sourceColor = tinycolor(color);
+    const palette = [...new Array(amount)].map((_, index) => {
+        const name = (90 - index * 10).toString();
+        const lightColor = sourceColor.lighten(index * 3).toRgb();
+        return {
+            ...lightColor,
+            name,
+        };
+    });
+    return palette;
+};
+
+export const EXAMPLE_PALETTES: Palette[] = [
```

Comment:
```suggestion
const EXAMPLE_PALETTES: Palette[] = [
```
Otherwise on Storybook 
<img width="224" alt="Capture d’écran 2022-01-21 à 09 59 40" src="https://user-images.githubusercontent.com/6846950/150497841-09a1dd21-dfd8-4cfd-af9f-7049a3749b8e.png">

Reply:
Moved it to its own file now :)